### PR TITLE
Honor {{ ansible_ssh_key }} var when copying keys

### PR DESCRIPTION
**Pull request by smalleni**
Signed-off-by: Sai Sindhur Malleni <smalleni@redhat.com>

### CODE REVIEW

File: tasks/copykeys.yml [hunk 1]
```diff
@@ -1,15 +1,15 @@
 ---
 - name: copy jump host key
   shell: |
-    echo '{{ ansible_ssh_pass }}' | sshpass ssh-copy-id  -o 'StrictHostKeyChecking no' -f {{ ssh_user }}@{{ hostname }}
+    echo '{{ ansible_ssh_pass }}' | sshpass ssh-copy-id -i {{ ansible_ssh_key }} -o 'StrictHostKeyChecking no' -f {{ ssh_user }}@{{ hostname }}
```
Comment (venkataanil) [inline tasks/copykeys.yml:4]: Why is this needed?

Comment (smalleni) [reply tasks/copykeys.yml:4]: Otherwise it always copies the user’s default keys and does not honor if you give a custom ansible_ssh_key var. right now this always works since it just copied the default keys and we don’t give it a custom path for ansible_ssh_key

Comment (venkataanil) [reply tasks/copykeys.yml:4]: Thanks
